app/views/hyku_addons/current_he_institution/_show.html.erb

```diff
@@ -1,8 +1,8 @@
-<!-- rendered from views/hyrax/base///_attribute_rows.html.erb -->
-<% array_of_hash = get_model(presenter.current_he_institution, presenter.model, 'current_he_institution', nil) %>
+<% if presenter.respond_to?(:current_he_institution) %>
```

Comment:
Is this guard necessary since it was also added to the render in `_attribute_rows.html.erb`